src/components/Atoms/Select/index.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import MenuItem from '@mui/material/MenuItem'
+import FormControl from '@mui/material/FormControl'
+import { Select, SelectProps } from '@mui/material'
+import { Iprops } from '../../../utils/Constants'
+
+const CustomSelect: React.FC<Iprops & SelectProps> = (props) => {
+  const { items, value, handleChange } = props
+  const menuItems = items.map((item) => (
+    <MenuItem key={item.id} value={item.id}>
+      {item.content}
+    </MenuItem>
+  ))
+
+  return (
+    <FormControl fullWidth size="small">
```

Comment:
Styles should be configurable.
If hardcoded as "small", it can be used in other places if medium or large Select is required